### ISSUE TITLE
iCloud account: accept all feeds in local (sidebar-originating) drags — don’t run feed finder code

### ISSUE THREAD

**Issue by brentsimmons** [Ranchero-Software/NetNewsWire#2888]
Yes, this could mean we end up with feeds that aren’t feeds, and we won’t have alerted the user. (We will be adding a Dinosaurs feature for that in a post-6.0 version of NetNewsWire.)

The important thing is that this will make for a far better initial experience than the one where a user gets a bunch of notifications about not being able to find a feed, and then they can’t be sure if they can delete from their originating account without a bunch of checking.

**Comment by WanderingStar** [comment]
I tried this again in Version 6.0b3 (6023)
It worked better. The process completed and my folders are populated, but I still got several of these:

<img width="342" alt="image" src="https://user-images.githubusercontent.com/1622203/111558067-02e16900-875c-11eb-969a-012d2613a206.png">

**Comment by brentsimmons** [comment]
Had you dragged feeds entirely within the sidebar, from one account to another? Or were you adding feeds by some other method?

**Comment by WanderingStar** [comment]
I dragged them from the "On My Mac" account to the "iCloud" account.

If it might make any difference, the iCloud account wasn't completely fresh. I did this before (see issue #2887). I'd deactivated the iCloud account (with the checkbox in Preferences) in 6.0b2. I then installed 6.0b3 (from a .zip), re-activated the iCloud account, selected all of the feeds from the previous drag-and-drop, deleted them (confirmation dialog), selected all of the feeds from On My Mac and dragged them all to iCloud. I didn't count the "The feed couldn't be found and can't be added" dialogs, but it was 3-5. When I tried this in 6.0b2, I think I had more dialogs, and some of the dialogs listed a large number of feeds that couldn't be added.
